docs/compression_algorithms/Quantization.md

```diff
@@ -131,8 +131,10 @@ There is a known issue with AVX2 and AVX512 CPU devices. The issue appears with
 AVX2 and AVX512 utilize a 16-bit register to store the result of operations on tensors. In case when tensors are saturated the buffer overflow happens.
 This leads to accuracy degradation.
 
-To fix this issue inside NNCF, weight tensors are quantized in 8 bits but only 7 bits are effectively used.
-This regime is used when `"target_device": "CPU"` or `"target_device": "ANY"` set.
+To fix this issue inside NNCF, by default, all weight tensors are quantized in 8 bits but only 7 bits are effectively used.
+This regime is used when `"target_device": "CPU"` or `"target_device": "ANY"` set. This fix, potentially, requires longer fine-tuning.
+
+To control the application of saturation fix, `"saturation_fix"` config option is introduced. The default value is `"saturation_fix": "enable"`. To apply the saturation issue fix only to the first layer, use `"saturation_fix": "enable_for_first_conv_layer"`. To disable the saturation issue fix for all layers, use `"saturation_fix": "disable"`.
```

Comment:
Several comments. In non-OV literature this problem is called overflow. I noticed that people could get it more easily. I think we could also provide a link to where users can read about this problem in details, e.g. [here ](https://www.intel.com/content/www/us/en/developer/articles/technical/lower-numerical-precision-deep-learning-inference-and-training.html).

Given that, I suggest rename it to overflow_fix. I also suggest renaming the second option to 
"first_layer_only".